docs/companion.md

```diff
@@ -457,6 +457,10 @@ The name of the bucket to store uploaded files in.
 
 The datacenter region where the target bucket is located.
 
+##### `COMPANION_AWS_PREFIX`
```

Comment:
Is it correct that this is a standalone only feature? It's not possible to set a prefix when integrating into Express?

Reply:
Ah yes, it's standalone only. But the same can be achieved with a custom `getKey` - let me add that.